miq_scripts/run.rb

```diff
@@ -1,4 +1,13 @@
-print "Assinging alert profiles to the Enterprise\n"
+puts "Assinging alert profiles to the Enterprise"
 
 MiqAlertSet.find_by(:guid=>"a16fcf51-e2ae-492d-af37-19de881476ad").assign_to_objects(MiqEnterprise.last)
 MiqAlertSet.find_by(:guid=>"ff0fb114-be03-4685-bebb-b6ae8f13d7ad").assign_to_objects(MiqEnterprise.last)
+
+puts "Enabling C&U roles"
+
+server_config = MiqServer.my_server.get_config()
+roles = server_config.config[:server][:role].split(',')
+roles |= ['ems_metrics_collector','ems_metrics_coordinator','ems_metrics_processor']
+roles = roles.sort().join(',')
+MiqServer.my_server.set_config(:server=>{:role => roles})
+MiqServer.my_server.save()
```

Comment:
Please verify that configuring C & U prior adding any provider actually function as we expect it to.
In case it's not than the above 2 actions need to be separated

Reply:
I'm pretty sure it's independent.  MiqServers have roles.  Then the when MiqWorkers "want to run" (*) on a specific server, they check if server roles allow it.

(*) by this anthropomorphism I mostly mean various `sync_workers` methods.
Some are independent of providers; some like `PerEmsWorkerMixin` want to run as result of adding a provider.  And I think specifically metric workers are constant, not per-ems.

Reply:
I tested it and it seemed to work